.storybook/main.js

```diff
@@ -9,9 +9,14 @@ module.exports = {
     '@storybook/addon-a11y',
     '@storybook/addon-actions',
     '@storybook/addon-viewport',
+    '@storybook/addon-interactions',
```

Comment:
neat, https://storybook.js.org/addons/@storybook/addon-interactions

Reply:
Ya, this could be super useful long term